DedicatedServer/Scripts/Libs/Zrx/ModeLibs/FlagRush/FlagRush_Map.Script.txt

```diff
@@ -53,7 +53,7 @@ CMapLandmark[] GetBases(Integer Clan) {
  * Chooses a random flag spawn.
  */
 CMapLandmark GetRandomFlagSpawn() {
-	declare Integer RandomIndex = ML::Rand(0, G_FlagSpawns.count - 1);
+	declare Integer RandomIndex = ML::Rand(0, G_FlagSpawns.count - 1, Now);
```

Comment:
Is the last parameter the seed or what does it do?

Reply:
IT's for seed, problem currently is it uses same seed everytime, and ppl have figured out the map spawns at same locations one after one, so the players who knows the maps very good have advantage since they know where the flag spawns next, using gametime as seed should resolve this problem.